invenio/modules/formatter/engine.py

```diff
@@ -2018,7 +2018,7 @@ def __init__(self, recID, ln=CFG_SITE_LANG, search_pattern=None,
             # If record is given as parameter
             self.xml_record = xml_record
             self.record = create_record(xml_record)[0]
-            recID = int(record_get_field_value(self.record, "001")) or None
+            recID = record_get_field_value(self.record, "001") or None
```

Comment:
Since you introduced this in 947910cd2, can you prepare test case for it? Thanks